### DISCUSSION TITLE
checkbox and combobox reset multisect of the datagrid

### DISCUSSION THREAD

**Discussion by STRUGAT** [AvaloniaUI/Avalonia#13516] [Q&A]
If you select several rows in the datagrid and then click on the checkbox or open a combobox, the selection of other rows will be reset. How to make the selection not reset?

**Comment by timunie** [comment]
idea: 

```cs
DataGrid.AddHandler(PointerReleasedEvent, Handler, RoutingStrategies.Tunnel);

private void Handler(object? sender, PointerPressedEventArgs e)
{
    e.Handled = To check if ComboBox was the source;
}
```

**Reply by timunie** [reply]
not sure about correct event though. Idea shold somehow work anyway.

**Comment by timunie** [comment]
Okay found we have `CellPointerPressed`-event which exposes the PointerPressedEvent. However if this event is marked as handled, we should maybe not select the row. Seems to be a bug to me. Can you make an issue of it with detail usecase?
